### DISCUSSION TITLE
Playback Reporting Plugin? Does it pull history before Jellystat is installed?

### DISCUSSION THREAD

**Discussion by sbsaylors** [CyferShepard/Jellystat#241] [Q&A]
Absolutely love this plugin. Brilliantly done really. Here's my question for help. 

Not seeing history of before setting up Jellystat reflected?  

Jellyfin in docker on single host, all standard default settings, about a year ago.
Jellystat in docker on same single host, all standard defaut settings, just a few days ago.
Jellyfin API setup in the docker-compose and things are working, just not from the "Import Playback Reporting Plugin Data" it seems.

I only see stats in Jellystat - starting from the day I installed it?  

I have clicked the "Import Playback Reporting Plugin Data" and it says it's running - and no errors, but never see any of the stats from before installing Jellystat.

Any thought how to trouble shoot?

**Comment by ljlzzzzz** [comment]
try restart jellystat, i use this way can get history data

**Reply by sbsaylors** [reply]
Restarted dozens of times. Updated to 1.1.2, tried re-import, no errors. Says successful. Still no data past the first install of Jellystat. Maybe next version. I could re-create the Db but it's working and updating, just not the import part.

**Comment by CyferShepard** [comment]
Hey Everyone, i plan to get around to this bug. Since iv changed how the release cycle works to push out faster updates, id recommend checking the release notes to see what changes and bug fixes where added. Il also post back here once the fix is ready.
